### PR TITLE
I18n prep

### PR DESCRIPTION
**Pull request by keberlin**

### CODE REVIEW

File: kano_profile_gui/QuestList.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def __init__(self, win, quest_info):
         quest_icon.set_margin_bottom(10)
 
         # Get text for the reward_list_widget
-        reward_text = "Rewards: "
+        reward_text = N_("Rewards: ")
```
Comment (tombettany) [inline kano_profile_gui/QuestList.py]: `reward.title` gets appended to this - is that marked for translation too?

File: kano_profile_gui/QuestList.py [hunk 2]
```diff
@@ -190,31 +190,31 @@ def __init__(self, win, quest_info):
...
+            self.background.get_style_context().remove_class('hover')
 
     def claim_reward(self, widget=None, event=None):
         self.quest_info.mark_completed()
 
         # TODO: this feels quite hacky. Should be moved back to the MenuBar
         # class, and maybe use signals instead?
         menu_bar = self.win.menu_bar
-        quest_button = menu_bar.get_button("QUESTS")
+        quest_button = menu_bar.get_button('QUESTS')
```
Comment (tombettany) [inline kano_profile_gui/QuestList.py]: Should this be the translated version?

File: kano_world/session.py [hunk 3]
```diff
@@ -299,7 +299,7 @@ def backup_content(self, file_path):
         except IOError as e:
             # There was an error somewhere, do not upload files
             files = {}
-            text = 'Error opening 1 of the files to backup {}'.format(str(e))
+            text = "Error opening 1 of the files to backup {}".format(str(e))
```
Comment (tombettany) [inline kano_world/session.py]: Are these customer-facing messages?

File: kano_world/session.py [hunk 4]
```diff
@@ -369,13 +369,13 @@ def upload_share(self, file_path, title, app_name):
...
                         logger.debug(
-                            'uploading {}: {}'.format(key, attachment_path))
+                            "uploading {}: {}".format(key, attachment_path))
                         files[key] = open(attachment_path, 'rb')
                         break
 
         except IOError as e:
             files = None
-            txt = 'Error opening the files to be shared {}'.format(str(e))
+            txt = "Error opening the files to be shared {}".format(str(e))
```
Comment (tombettany) [inline kano_world/session.py]: And this one?

File: kano_profile/profile.py [hunk 5]
```diff
@@ -203,14 +203,14 @@ def set_avatar(subcat, item, sync=False):
...
-    "Video Set": "video_set",
-    "Minecraft": "minecraft"
+    _("Arcade Hall"): 'arcade_hall',
+    _("Concert Hall"): 'concert_hall',
+    _("Jungle"): 'jungle',
+    _("Space"): 'space',
+    _("Dojo"): 'dojo',
+    _("Park"): 'park',
+    _("Video Set"): 'video_set',
+    _("Minecraft"): 'minecraft'
```
Comment (tombettany) [inline kano_profile/profile.py]: The keys here are being translated but the [`get_environment()`](https://github.com/KanoComputing/kano-profile/blob/650d21589eb91ba44a18114511393e0e964c3136/kano_profile/profile.py#L222-L230) function is retrieving the environments from the saved profile (which will and should be in English) and comparing them with the `ENVIRONMENT_REMAP` keys which is likely to fail.

**Comment by keberlin** [comment]
Hi Tom,

Not entirely sure about the logger. ones in session.py? Is this a debugging
logger? Or appearing on the screen?

Keith

File: kano_login/login.py [hunk 6]
```diff
@@ -310,12 +310,12 @@ def __init__(self, win):
 
         self.email_entry = self.labelled_entries.get_entry(0)
         self.email_entry.set_text(user_email)
-        self.email_entry.connect("key-release-event", self.activate)
+        self.email_entry.connect('key-release-event', self.activate)
 
-        self.button = KanoButton(_("Reset password").upper())
+        self.button = KanoButton(_("RESET PASSWORD"))
```
Comment (radujipa) [inline kano_login/login.py]: As a note: removing the `.upper()` from here assumes the translators will know to keep the styling..

File: kano_registration_gui/RegistrationScreen.py [hunk 7]
```diff
@@ -105,43 +105,43 @@ def _on_register_button(self, widget=None):  # TODO: refactor this
         # screen. However there is a small chance someone could take the
         # username while the user is in the process of registering
         if not success:
-            if text.strip() == "Cannot register, problem: " \
-               "Username already registered":
+            if text.strip() == _("Cannot register, problem: " \
```
Comment (radujipa) [inline kano_registration_gui/RegistrationScreen.py]: So.. this `text` is coming from `kano_world.functions`, which returns now also a translated string, but which gets formatted by whatever `request_wrapper` will return.. And so on.. Need to be careful when testing this one..

File: kano_registration_gui/RegistrationScreen.py [hunk 8]
```diff
@@ -155,7 +155,7 @@ def _is_username_available(self, name):
             headers=content_type_json
         )
 
-        if not success and text.strip() == "User not found":
+        if not success and text.strip() == _("User not found"):
```
Comment (radujipa) [inline kano_registration_gui/RegistrationScreen.py]: Similar note here from `register`

File: kano_world/session.py [hunk 9]
```diff
@@ -76,11 +76,11 @@ def upload_profile_stats(self):
...
         try:
             gender = profile['gender']
-            if gender == 'Boy':
+            if gender == _("Boy"):
                 gender = 'm'
-            elif gender == 'Girl':
+            elif gender == _("Girl"):
                 gender = 'f'
-            elif gender == "Wizard":
+            elif gender == _("Wizard"):
```
Comment (radujipa) [inline kano_world/session.py]: Just wondering how this is going to impact our data tracking.. Is @delormev going to see the Spanish word for "Wizard"?

**Comment by rcocetta** [comment]
What is the situation with this pull request?

**Comment by tombettany** [comment]
@rcocetta I've started to look into this one but haven't got much to show as of yet.

**Comment by tombettany** [comment]
I've made some changes which will hopefully fix a lot of the concerns. Will merge now so that we can get properly testing tomorrow.

**Pull request merged**